packages/phone-number-privacy/combiner/src/match-making/get-contact-matches.ts

```diff
@@ -47,7 +47,17 @@ export async function handleGetContactMatches(request: Request, response: Respon
       respondWithError(response, 400, WarningMessage.INVALID_INPUT, logger)
       return
     }
-    if (!(await authenticateUser(request, getContractKit(), logger))) {
+
```

Comment:
is there any issue if no error is thrown? Or does this just then return an empty list of contact matches?

Reply:
If I understand correctly you're asking if there's an issue with catching the error here even though the check was not succesful? We catch the error and assume the user is authenticated to provide better onboarding UX when there are availability issues. I.e. we "fail open".

Reply:
I mean does it cause any confusing behavior later if we don't throw an error here? So from your response, is this correct: we continue as if everything is working as expected at this point, log the error, and at the very end we would return an empty list of matches to allow for a cleaner onboarding flow (as a temp solution) right? Just trying to understand if there's any issues with _not_ throwing an error here! Thanks 🙏